src/main/java/io/strimzi/kafka/bridge/tracing/OpenTelemetryHandle.java

```diff
@@ -70,7 +70,7 @@ public String serviceName(BridgeConfig config) {
     @Override
     public void initialize() {
         System.setProperty("otel.metrics.exporter", "none"); // disable metrics
-        System.setProperty("otel.logs.exporter", "none"); // disable logs
+        System.setProperty("otel.logs.exporter", "none"); // disable otel logs exporter. This has no effect on logging in the bridge.
```

Comment:
I know it's not related to this PR but you expand the comment about "disable metrics" the same way for consistency please? Something like you have for logs, i.e. `// disable otel metrics exporter`. It sounds useless opening a different PR just for that. Thanks.

Reply:
resolved with https://github.com/strimzi/strimzi-kafka-bridge/pull/867/commits/14e150d7a0104c235e2094a57afec185d535c19d